ffi/src/debug.rs

```diff
@@ -0,0 +1,49 @@
+pub use self::implementation::*;
+
+#[cfg(feature = "debug_mode")]
+mod implementation {
+    use std::fs::OpenOptions;
+    use std::sync::Once;
+
+    use tracing_subscriber::prelude::*;
+    use tracing_subscriber::EnvFilter;
+
+    static SETUP: Once = Once::new();
+
+    pub fn setup_logger() {
+        SETUP.call_once(|| {
+            let path = format!("{}/sspi-rs.log", std::env::var("HOMEPATH").unwrap());
```

Comment:
As a comment, `HOMEPATH` might not be the best name for the environment variable (I could envisage a conflict). Maybe something SSPI_* (like the other environment variables in the project)

Reply:
Indeed, `HOMEPATH` is the current user’s home path on Windows, it was not meant to be overridden. I agree we should add alternatives with the `SSPI_` prefix! And fallback wih `HOME` on Linux and MacOS (currently I’m pretty sure it will panic :sweat_smile: )

Reply:
Ok great; I'm debugging on Android where it must be overridden - $HOME on Android and iOS is outside the application sandbox and not writeable.

I just tested this on Android and interestingly, it panics on the second `unwrap` (creating the file). I do not know why 🤷 If I create the file myself (programmatically, on the managed side) before bootstrapping the library, it does work (I get the detailed error written to sspi-rs.log).  So I'm pretty sure this is not a permissions issue (noting also that the directory I'm using is definitely writeable from native code - I have FreeRDP writing its logs there too). Very confusing but I have a workaround for now.